tools/ete/ete_init_taxdb.py

```diff
@@ -0,0 +1,26 @@
+import optparse
+# import sys
+# sys.path.append("./ete/")
+
+import ete3.ncbi_taxonomy
+
+parser = optparse.OptionParser()
+parser.add_option('-t', '--taxdump', dest='taxdump', default=None,
+                  help='NCBI taxdump (tar.gz) will be downloaded if not given')
+parser.add_option('-d', '--database', dest="database", default=None,
+                  help='ETE sqlite data base to use (default: ~/.etetoolkit/taxa.sqlite)')
+options, args = parser.parse_args()
+if options.database is None:
+    parser.error("-d option must be specified")
+if options.taxdump is not None:
+    taxdump = options.taxdump
+else:
+    from urllib import urlretrieve
```

Comment:
For Python3 compatibility, can you please change this to:
```
from six.moves.urllib.request import urlretrieve
```
and move it to the top, just after the `ete3.ncbi_taxonomy` import?